src/main/java/org/fundacionjala/pivotal/pageobjects/login/HomePage.java

```diff
@@ -15,6 +15,7 @@
     //Page URL
     @FindBy(how = How.LINK_TEXT, using = "Sign in")
     private WebElement signin;
+    //TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)